generators/app/index.js

```diff
@@ -108,6 +108,24 @@ module.exports = JhipsterGenerator.extend({
             }.bind(this));
         },
 
+        checkNode: function () {
+            if (this.skipChecks || this.skipServer) return;
+            var done = this.async();
+            exec('node -v', function (err, stdout, stderr) {
+                if (err) {
+                    this.warning('Node is not found on your computer.');
+                } else {
+                    var nodeVersionMajor = stdout.split('.')[0].replace(/v/g, '');
+                    var nodeVersionMinor = stdout.split('.')[1];
+                    var nodeVersion = stdout.replace(/\n/g, '');
+                    if (nodeVersionMajor < 6 || (nodeVersionMajor >= 6 && nodeVersionMinor < 9)) {
```

Comment:
Btw if we are checking this then better to use the semver library to do that we already use that  in `isJhipsterVersionLessThan` method in `generator-base.js`